### PR TITLE
OSDOCS-9631: adds the bug fix relnotes Microshift 4.15

### PR DESCRIPTION
**Pull request by aravipra**
Version(s):
4.15 only

Issue:
[OSDOCS-9631](https://issues.redhat.com/browse/OSDOCS-9631)

Link to docs preview:
[Asynchronous errata updates](https://71644--ocpdocs-pr.netlify.app/microshift/latest/microshift_release_notes/microshift-4-15-release-notes#microshift-4-15-asynchronous-errata-updates)

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

File: microshift_release_notes/microshift-4-15-release-notes.adoc [hunk 1]
```diff
@@ -197,7 +197,7 @@ In the following tables, features are marked with the following statuses:
 [id="microshift-4-15-asynchronous-errata-updates"]
 == Asynchronous errata updates
 
-Security, bug fix, and enhancement updates for {microshift-short} {product-version} are released as asynchronous errata through the Red Hat Network. All {microshift-short} {product-version} errata are https://access.redhat.com/downloads/content/290/[available on the Red Hat Customer Portal]. Read the https://access.redhat.com/support/policy/updates/openshift[{microshift-short} Life Cycle] for more information about asynchronous errata.
+Security, bug fix, and enhancement updates for {microshift-short} {product-version} are released as asynchronous errata through the Red Hat Network. All {microshift-short} {product-version} errata are https://access.redhat.com/downloads/content/290/[available on the Red Hat Customer Portal]. Read the https://access.redhat.com/product-life-cycles?product=Red%20Hat%20build%20of%20Microshift,Red%20Hat%20Device%20Edge[{microshift-short} Life Cycle] for more information about asynchronous errata.
```
Comment (lpettyjo) [inline microshift_release_notes/microshift-4-15-release-notes.adoc:200]: Move the intro statement to the front of the sentence to let readers know if they are interested in the link: "For more information about asynchronous errata, read the https://access.redhat.com/product-life-cycles?product=Red%20Hat%20build%20of%20Microshift,Red%20Hat%20Device%20Edge[{microshift-short} Life Cycle]."

**Review by lpettyjo — COMMENTED** [review]
LGTM!

File: microshift_release_notes/microshift-4-15-release-notes.adoc (hunk 1 above)
Comment (aravipra) [reply microshift_release_notes/microshift-4-15-release-notes.adoc:200]: Done

**Comment by aravipra** [comment]
QE approval not required as no technical changes are made and it is internal doc work

**Pull request merged**